provisioners/file/home/ec2-user/bin/limit-logs

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+cd /data/jenkins/jobs
+sudo find . -type f -name '*log' -size +15M \
```

Comment:
Why is this running as `ec2-user` and then needs sudo?

Wouldn't it be better to run this as a root cron job?

Reply:
This is just personal preference, but I prefer

```
find /dir ...
```

over

```
cd /dir
find . ...
```

It doesn't matter for a trivial script like this, but the former version remains valid even when the cwd is changed elsewhere.

Reply:
Sure, can make that change. Re the sudo, see Mark's comment and my answers.

Reply:
Github never showed me Mark's comments when I added mine; otherwise I wouldn't have bothered 😄

Reply:
Changed with commit 5f4ac1ddbc5349bd9d607afb808001070b44de43